project/scalapb.sbt

```diff
@@ -0,0 +1,3 @@
+libraryDependencies ++= Seq(
+  "com.github.os72" % "protoc-jar" % "3.0.0-b1"
```

Comment:
This will not propagate too the submodules. I verified that it does not end up in submodule's `libraryDependencies`. Initially I wanted to put it in `scioSchema` but then I get failure:

```
scio/build.sbt:313: error: object os72 is not a member of package com.github
    com.github.os72.protocjar.Protoc.runProtoc("-v300" +: args.toArray)
```

when I try to reference it in subsequent `settings` section.